misk-admin/src/main/kotlin/misk/web/dashboard/MiskWebTabIndexAction.kt

```diff
@@ -0,0 +1,91 @@
+package misk.web.dashboard
+
+import kotlinx.html.body
+import kotlinx.html.div
+import kotlinx.html.head
+import kotlinx.html.html
+import kotlinx.html.id
+import kotlinx.html.link
+import kotlinx.html.p
+import kotlinx.html.script
+import kotlinx.html.style
+import misk.hotwire.buildHtml
+import misk.web.Get
+import misk.web.PathParam
+import misk.web.ResponseContentType
+import misk.web.actions.WebAction
+import misk.web.mediatype.MediaTypes
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Kotlin backed tab loader, equivalent to /_tab/slug/index.html
+ */
+@Singleton
+class MiskWebTabIndexAction @Inject constructor(
+  @AdminDashboard private val dashboardTabs: List<DashboardTab>,
+) : WebAction {
+  @Get("$PATH/{slug}")
+  @ResponseContentType(MediaTypes.TEXT_HTML)
+  @AdminDashboardAccess
+  fun get(@PathParam slug: String?): String {
+    val dashboardTab = dashboardTabs.firstOrNull { slug == it.slug }
+    return buildHtml {
+      html {
```

Comment:
This returns a Kotlin-ify'd version of the auto-generated `/_tab/<slug>/index.html` that Misk-Web's Webpack currently pops out. Putting this in Kotlin makes it easier to apply universal styles across all tabs being loaded.